Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Parent Dockerfile https://github.com/docker-library/mongo/blob/982328582c74dd2f0a9c8c77b84006f291f974c3/3.0/Dockerfile
+FROM mongo:3.4
+
+# Modify child mongo to use /data/db2 as dbpath (because /data/db wont persist the build)
+RUN mkdir -p /data/db2 \
+    && echo "dbpath = /data/db2" > /etc/mongodb.conf \
+    && chown -R mongodb:mongodb /data/db2
+
+COPY data/ /pharmacy-data/
+
+RUN mongod --fork --logpath /var/log/mongodb.log --dbpath /data/db2 --smallfiles \
```

Comment:
If `mongodb.conf` is contained in the repo this line could change to just `RUN mongod --config etc/mongod.conf` (I think all options are possible to have set in the config). Although you might need to keep the `--fork` option out of the config so it can be used later when the process is started in the foreground.